consensus/hotstuff/committees/cluster_committee.go

```diff
@@ -0,0 +1,104 @@
+package committees
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/consensus/hotstuff"
+	"github.com/onflow/flow-go/consensus/hotstuff/committees/leader"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/model/flow/filter"
+	"github.com/onflow/flow-go/state/protocol"
+	"github.com/onflow/flow-go/storage"
+)
+
+// Cluster represents the committee for a cluster of collection nodes. Cluster
+// committees are epoch-scoped.
+//
+// Clusters build blocks on a cluster chain but must obtain identity table
+// information from the main chain. Thus, block ID parameters in this Committee
+// implementation reference blocks on the cluster chain, which in turn reference
+// blocks on the main chain - this implementation manages that translation.
+type Cluster struct {
+	state    protocol.ReadOnlyState
+	payloads storage.ClusterPayloads
+	me       flow.Identifier
+	// pre-computed leader selection for the full lifecycle of the cluster
+	selection *leader.LeaderSelection
+	// initial set of cluster committee members for this epoch, used only for
+	// mapping a leader index to node ID
+	// CAUTION: does not contain up-to-date weight/ejection info
+	identities flow.IdentityList
```

Comment:
would suggest to move (as `IdentifierList`) into `LeaderSelection`

Reply:
done